waiter/src/waiter/scheduler/kubernetes.clj

```diff
@@ -469,7 +472,7 @@
                               :log-directory (log-dir-path run-as-user primary-container-restart-count)
                               :port port0
                               :service-id service-id
-                              :started-at pod-started-at
+                              :started-at (or main-container-started-at main-container-last-terminated-at pod-started-at)
```

Comment:
A slight improvement over the previous PR to include the last termination time if the container hasn't transitioned to running yet.